test/integration/middleware/core/test/index.test.js

```diff
@@ -212,6 +212,24 @@ function rewriteTests(locale = '') {
     expect($('.title').text()).toBe(expectedText)
   })
 
+  it(`${locale} should clear query parameters`, async () => {
```

Comment:
Can we add a test case to ensure the warning is shown?

Reply:
done